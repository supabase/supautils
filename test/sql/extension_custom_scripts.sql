set role extensions_role;
\echo

-- per-extension custom scripts are run
create extension fuzzystrmatch;
drop extension fuzzystrmatch;
select * from t2;

reset role;
drop table t2;
set role extensions_role;
\echo

-- global extension custom scripts are run
create extension dict_xsyn;
create extension insert_username version "1.0" schema public cascade;
\echo

-- custom scripts are run even for superusers
reset role;
create extension fuzzystrmatch;
drop extension fuzzystrmatch;
select * from t2;

drop table t2;
set role extensions_role;
\echo
