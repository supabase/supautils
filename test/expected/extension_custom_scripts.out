set role extensions_role;
\echo

-- per-extension custom scripts are run
create extension fuzzystrmatch;
drop extension fuzzystrmatch;
select * from t2;
 column1 
---------
       1
(1 row)

reset role;
drop table t2;
set role extensions_role;
\echo

-- global extension custom scripts are run
create extension dict_xsyn;
NOTICE:  extname: dict_xsyn, extschema: <NULL>, extversion: <NULL>, extcascade: f
create extension insert_username version "1.0" schema public cascade;
NOTICE:  extname: insert_username, extschema: public, extversion: 1.0, extcascade: t
\echo

-- custom scripts are run even for superusers
reset role;
create extension fuzzystrmatch;
drop extension fuzzystrmatch;
select * from t2;
 column1 
---------
       1
(1 row)

drop table t2;
set role extensions_role;
\echo

